concepts/strings/about.md

```diff
@@ -24,7 +55,13 @@ Some values need to be escaped:
 | `\"`  | Double quote         |
 
 ```go
-const daltons = "Joe\nWilliam\nJack\nAverell"
+fmt.Println("Joe\nWilliam\nJack\nAverell") 
+
+// Output:
+//Joe
+//William
+//Jack
+//Averell
 ```
 
 Raw string literals use backticks (\`) as their delimiter instead of double quotes and are interpreted literally, meaning that there is no need to escape characters or newlines.
```

Comment:
```suggestion

## Raw string literals

Raw string literals use backticks (\`) as their delimiter instead of double quotes and all characters in it are interpreted literally, meaning that there is no need to escape characters or newlines.
```

Apart from the example given here, I think it would be useful to include another one where special characters like `\n`are used to show the difference. We do something similar in the [regular expressions](https://github.com/exercism/go/blob/main/concepts/regular-expressions/introduction.md) concept where a regular double-quoted string literal is compared to a raw string literal:

```go
"\t\n" // regular string literal with 2 characters: a tab and a newline
`\t\n`// raw string literal with 4 characters: two backslashes, a 't', and an 'n'
```

You can think of a better example, but feel free to use the same example as the one in that concept if you want. Since this is relevant in both places, I don't mind a little bit of repetition.